logger.go

```diff
@@ -307,6 +307,42 @@ func (l *Logger) V(lvl Level) Verbose {
 	}
 }
 
+// AddInfoLogOutPut add output destination for info Logger
+func (l *Logger) AddInfoLogOutPut(writers ...io.Writer){
```

Comment:
AddInfoLogOutput, output is one word (here and below)

Reply:
fixed